doc/source/rllib/rllib-algorithms.rst

```diff
@@ -70,8 +70,8 @@ Algorithm                     Frameworks Discrete Actions        Continuous Acti
 .. _`+RNN`: rllib-models.html#rnns
 .. _`TD3`: rllib-algorithms.html#ddpg
 .. _`+Attention`: rllib-models.html#attention
-.. _`TS`: rllib-models.html#lints
-.. _`UCB`: rllib-models.html#linucb
+.. _`TS`: rllib-algorithms.html#lints
```

Comment:
why did you change the `TS` link? Doesn't look right to me.

Reply:
`TS` link before was not pointing to the Thompson Sampling algorithm before.  Now it points there.